CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 9.0.0
+
+* BREAKING: Enable production-like logs if env variable is set ([#302](https://github.com/alphagov/govuk_app_config/pull/302))
```

Comment:
Not sure I fully understood what you mean by "production-like" logs. Is it switch the logs to JSON formatting etc?

Reply:
@theseanything I think Kevin's comment above explains this best. Yes, it switches the logs to JSON - the main chunk of work is actually in #300

Reply:
(Sorry just realised Kevin left a similar comment)